scripts/ci/make/util

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+#########################################################
+###   COMMON UTILITY FUNCTIONS - ALPHABETICAL ORDER   ###
+#########################################################
+
+cleanup() {
+    rm -rf "${VENV}/tmp"
+}
+
+clean_container() {
+    print_colorized ERROR "deleting ${TEST_IMAGE_NAME}"
+    sleep 3
+    docker rmi "${TEST_IMAGE_NAME}"
+}
+
+clean_python_env() {
+    print_colorized ERROR "deleting virtualenv & build data"
+    sleep 3
+    rm -rf "${VENV}" \
+        *.egg-info \
+        dist \
+        build
+    find . -name '*.pyc' -exec rm -f {} \;
```

Comment:
newer Python versions leave a `__pycache__` directory behind as well. One way I do this is with:

```bash
find . \( -type f -name "*.py[co]" -o -type d -name "__pycache__" \) -delete
```